{{cookiecutter.project_shortname}}/{{cookiecutter.python_package_manager == "uv" and "pyproject.toml" or "Pipfile"}}

```diff
@@ -0,0 +1,50 @@
+{%- if cookiecutter.python_package_manager == 'pipenv' -%}
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+check-manifest = ">=0.25"
+
+[packages]
+invenio-app-rdm = {extras = ["{{ cookiecutter.search }}"{% if cookiecutter.file_storage == 'S3' %}, "s3"{% endif %}], version = "~=13.0.0b2.dev2"}
+{%- if cookiecutter.site_code == 'yes'%}
+{{cookiecutter.project_shortname}} = {editable = true, path="./site"}
+{% endif %}
+uwsgi = ">=2.0"
+uwsgitop = ">=0.11"
+uwsgi-tools = ">=1.1.1"
+
+[requires]
+python_version = "3.9"
+
+[pipenv]
+allow_prereleases = false
+
+{%- endif -%}
+{%- if cookiecutter.python_package_manager == 'uv' -%}
+[project]
+name = "{{ cookiecutter.project_name | slugify() }}"
+description = "Add your description here"
+readme = "README.md"
+version = "0.1.0"
+requires-python = ">= 3.12"
+dynamic = ["version"]
+
+dependencies = [
+  "invenio-app-rdm[opensearch2]~=13.0.0b2.dev2",
```

Comment:
looks like the dynamic extras are missing here

perhaps it would make sense to calculate the extras in a `{%- set app_rdm_extras = ... -%}` block above, and simply reuse it in the `Pipfile` part and here?
similar for the app-rdm package version